src/calculateRentalCost.js

```diff
@@ -5,6 +5,23 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+
+  const longTerm = 7;
+  const shortTerm = 3;
+
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+
+  const total = days * cost;
+
+  if (days >= longTerm) {
+    return total - longTermDiscount;
+  } else if (days >= shortTerm) {
```

Comment:
Instead of using an `else if` after a return statement, you can remove the `else` to simplify the code. When the `if` condition is true and the return statement is executed, the function will exit, and the remaining code will not be executed anyway.